app/src/app/[lng]/data/[step]/page.tsx

```diff
@@ -417,6 +439,60 @@ export default function AddDataSteps({
 
   const [isDataSectionExpanded, setDataSectionExpanded] = useState(false);
 
+  const STATUS = "pending";
+  const URL = "http://localhost";
+
+  const getInventoryData = useSelector(
+    (state: RootState) => state.inventoryData,
+  );
+  const dispatch = useDispatch();
+  function fileToBase64(file: File) {
+    return new Promise((resolve, reject) => {
+      const reader = new FileReader();
+      reader.readAsDataURL(file);
+
+      reader.onload = () => resolve(reader.result);
+      reader.onerror = (error) => reject(error);
+    });
+  }
```

Comment:
Not sure if transmitting files as base64 is the best. It works for now, but transmitting it as a binary buffer (which imo is valid for form data) is probably better, especially for larger files.